templates/includes/sidebar.html

```diff
@@ -0,0 +1,8 @@
+<div class="sidebar">
+    <ul>
+        <li><a href="#">Home page</a></li>
+        <li><a href="#">Manufacturers</a></li>
+        <li><a href="#">Cars</a></li>
+        <li><a href="#">Drivers</a></li>

```

Comment:
The links currently use `#` as the href attribute, which is a placeholder. Consider replacing these with actual URLs or Django `{% url %}` template tags to link to the correct views.